parlai/core/pytorch_data_teacher.py

```diff
@@ -745,9 +769,13 @@ def act(self):
         return action
 
     def shutdown(self):
+        """Shut down."""
         super().shutdown()
         BatchSortCache.destroy()
 
 
 class DefaultTeacher(PytorchDataTeacher):
+    """Default teacher."""
+
+    # TODO: isn't having a DefaultTeacher here a bad idea? What is this used for?
```

Comment:
This is to mimic behavior of having a `DefaultTeacher` in other classes, i.e. so you do not have to specify `-t task:TeacherToUse`. It makes the logic easier to handle in the `get_task_module` function in `agents.py` (here: https://github.com/facebookresearch/ParlAI/blob/master/parlai/core/agents.py#L625)